app/portainer/components/Dashboard/DashboardItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Widget, WidgetBody } from '@/portainer/components/widget';
+
+interface Props {
+  value: number;
+  icon: string;
+  comment: string;
+  accessibilityLabel: string;
+}
+
+export function DashboardItem({
+  value,
+  icon,
+  comment,
+  accessibilityLabel,
```

Comment:
didn't we say we will remove this?

Reply:
removed the extra param, aria-label is now built from provided comment